conanfile.py

```diff
@@ -5,7 +5,7 @@
 
 class HomestoreConan(ConanFile):
     name = "homestore"
-    version = "4.0.11"
+    version = "4.0.12"
```

Comment:
We don't need to bump the version since there is no change to the library in this PR.